eth2near/eth2near-block-relay-rs/src/eth2near_relay.rs

```diff
@@ -74,11 +74,11 @@ impl Eth2NearRelay {
                 while headers.len() < self.max_submitted_headers as usize
                     && current_slot <= last_eth2_slot_on_eth_chain
                 {
-                    trace!(target: "relay", "Try add block header for slot={}, headers len={}/{}", current_slot, headers.len(), self.max_submitted_headers);
+                    debug!(target: "relay", "Try add block header for slot={}, headers len={}/{}", current_slot, headers.len(), self.max_submitted_headers);
                     let mut count = 0;
                     loop {
                         if count > 0 {
-                            trace!(target: "relay", "Error on extraction execution block header for slot = {}. Try again. Trying number {}", current_slot, count + 1);
+                            debug!(target: "relay", "Error on extraction execution block header for slot = {}. Try again. Trying number {}", current_slot, count + 1);
```

Comment:
Fixing an error message a little bit
```suggestion
                            debug!(target: "relay", "Error retrieving execution block header for slot = {}. Try again. Trying number {}", current_slot, count + 1);
```